ethereum-optimism/optimism
# feat(contracts-periphery): Add attestation contract

Add attestation contract
- Has `attest` to make attestations

Add attestation tests
- Added just two tests since they were my first ever tests
- Looking for feedback if tests are good and if I should add more

Update foundry.toml
- Had some issues here
- Add bedrock contracts since we use SemVer
- Add open zepplin contracts

Update package.json
- If I added latest contract bedrock the module got hoisted.  I couldn't target the node module when it was hoisted because foundry said it was outside of project
- Because of this the hack I did was use 0.10.0 instead of 0.11.0
- Open to a less hacky way of doing this
- Remove unused contracts-upgradeable

Deployment script
- Adding in a follow up pr
- Hacky deployment script we were using is here https://github.com/ethereum-optimism/gateway/blob/develop/packages/contracts/scripts/deploy.ts
